rkt/api_service.go

```diff
@@ -289,26 +283,27 @@ func getNetworks(p *pod) []*v1alpha.Network {
 func getBasicPod(p *pod) (*v1alpha.Pod, *schema.PodManifest, error) {
```

Comment:
It looks like the error returned here is never set, so it can be removed from the function signature.